build-fsal_ceph_and_rgw/build-fsal_ceph_and_rgw.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/sh
+
+set -e
+
+# these variables need to be set
+[ -n "${GERRIT_HOST}" ]
+[ -n "${GERRIT_PROJECT}" ]
+[ -n "${GERRIT_REFSPEC}" ]
+
+# only use https for now
+GIT_REPO="https://${GERRIT_HOST}/${GERRIT_PROJECT}"
+
+SHAMAN_REPO_URL="https://shaman.ceph.com/api/repos/ceph/master/latest/centos/7/flavors/default/repo"
```

Comment:
You should use `${CENTOS_VERSION}` instead of 7. CentOS-6 can be provided through the jobs too (and that should probably fail while installing dependencies).